core/templates/dev/head/pages/exploration_player/AnswerClassificationServiceSpec.js

```diff
@@ -24,7 +24,7 @@ describe('Answer classification service with string classifier disabled',
       module(function($provide) {
         $provide.constant('INTERACTION_SPECS', {
           RuleTest: {
-            is_string_classifier_trainable: false
```

Comment:
Careful. There are other instances of is_string_classifier_trainable in the backend codebase that also need to be replaced.